packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -35,22 +43,44 @@ export class ScDialSvg {
   @State() iconLeftTopX: number = 0;
   @State() iconLeftTopY: number = 0;
 
+  @State() label: string;
   @State() showLabel: boolean = false;
 
+  @State() defaultRing: string = '';
+  @State() ColorRing: string = '';
```

Comment:
use standard style for declaring variables, i.e. change from `ColorRing` -> `colorRing`

Reply:
Done